src/connection.js

```diff
@@ -33,12 +35,24 @@ var Connection = function( options, callback ) {
 Connection.prototype.get = function( recordId, callback ) {
   var value
   var params = this._getParams( recordId )
+
   this.client.get( {
     index: this._index,
     type: params.type,
     id: params.id
   }, ( error, response ) => {
-    if( error && error.displayName !== 'NotFound' ) {
+    if( error && response.error && response.error.type == 'index_not_found_exception' ) {
+      var body = { settings: {}, mappings: {} }
+
+      body.settings = JSON.parse(this._settings)
```

Comment:
Since settings don't change, might make sense to do parse within constructor for as small optmization